### ISSUE TITLE
Lion: After restart (set to re-open all apps on startup) tray, notification tray icon doesn't reappear.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#5]
```
What steps will reproduce the problem?
1. Restart a machine running Lion while retaining "Reopen windows when logging 
back in" checked.


What is the expected output? What do you see instead?
The notification tray icon will not turn on.  Re-installing the extension will 
bring it back, but that's not how it's supposed to work of course. :-)


What version of the product are you using? On what operating system?
Version 1.2.1

Please provide any additional information below.
Pretty straight-forward issue.  Hope this helps.

```

Original issue reported on code.google.com by `adrianjr...@gmail.com` on 31 Jul 2011 at 3:57

**Comment by GoogleCodeExporter** [comment]
```
I seem to be getting this problem too, my email is 
ryan_likestoread@hotmail.com. Could you email me how this can be avoided or how 
this can be fixed. Thank you. I truly love this app btw, its an extremely 
awesome little bit of programming!! :D
```

Original comment by `ryan.bro...@gmail.com` on 3 Nov 2011 at 5:45

**Comment by GoogleCodeExporter** [comment]
```
Happening to me, too. Is there a fix yet?
```

Original comment by `sarah.fe...@gmail.com` on 23 Jan 2012 at 12:21

**Comment by GoogleCodeExporter** [comment]
```
Also seeing this issue. Running Lion 10.7.3, and nosleep 1.2.1, MacBook Air.
```

Original comment by `jdonel...@gmail.com` on 20 Feb 2012 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
It will be fixed in 1.3
```

Original comment by `integral...@gmail.com` on 20 Feb 2012 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3
```

Original comment by `integral...@gmail.com` on 23 Feb 2012 at 8:39
- Changed state: **Fixed**
